util/Utility.py

```diff
@@ -0,0 +1,190 @@
+        JERIFISSHU = 208694891486773248
+        NSFW = 324699837297721344
+        JAZZ_ZONE = 335923711662227456
+        GAME_IDEAS = 842496144603873291
+        SHITPOST = 413881959857520640
+        DND = 790011375220162590
+        SET_AND_BETS = 580583079597572096
+        BUDS_N_CRAFT = 804897670065815552
+        BUD_COURT = 796190794334732298
+        NEW_CHANEL_OLD = 796187591162069003
+        CUM_CHANNEL = 819458956606439424
+        NEW_CHANNEL = 872613001709113414
+        MUSIC_QUEUE = 741139638072246323
+        GAMING_ZONE = 380181785390088203
+        DEBATE_ZONE = 786065089367113728
+        AFK = 291414646189981696
+        PINS = 789771971532947486
+        HALL_OF_FAME = 833063954602786836
+        ROBBOT_TESTING = 789662489092030494
+#endregion
+
+#region Channels
+'''
+@author: Keeth S.
+@desc: Enum of all Role ids
+'''
+class Roles(Enum):
+    PATRIARCH = 290567553510539264
+    CAPTAIN = 290566787580428288
+    DEV = 789929955743236106
+    LIEUTENANT = 290566792760262656
+    FUCKBOY = 323617141054111756
+#endregion
+
+#region Emojis
+'''
+@author: Keeth S.
+@desc: Enum of all Emoji ids
+'''
+class Emojis(Enum):
+    HEEHO = 324008832735117312
+    RISE = 324010369267793930
+    NEVERFELTLIKE = 324011042890055680
+    NOTLIKETHIS = 324012625342103552
+    AIGIS = 324013626253901836
+    FRANCISYORKMORGAN = 324013824338427905
+    HATSUNEMIKU = 324023809374158848
+    BLASTED = 324023892786151425
+    POSITIVETHINKING = 324023913946284035
+    EVERYDAY = 324694469150638093
+    STOP = 324694491510472704
+    OHWOW = 324694529733296128
+    KOROMARU = 324694541871480832
+    ORLANDO = 324694553200427009
+    SPEED = 324694586855391244
+    SUPERNUT = 324694596397432833
+    TIFA = 324694604903481344
+    VINCENT = 324694614823010314
+    BUTTHETEACHINGS = 324694624708984832
+    ZILTOID = 324694636972998657
+    AUTISTICSCREECHING = 324694662759841792
+    DERICK = 324695421639196672
+    HURR = 324696431904686090
+    CRAWLING = 324699598914453504
+    THATSRAD = 324706382781743105
+    STUPID_IDIOT = 324724786557353984
+    SMUGTEACHER = 324745655878811648
+    DDOG = 324750990630846474
+    SAKEGREMLIN = 324752939904270336
+    Q_ = 325104825882902538
+    DOTHEYSMUGEVERYTIME = 325105720121098240
+    HAHAA = 325106039345381376
+    KIRYU = 325106393701023744
+    LOLOL = 325107124931788800
+    LETSNOTDOTHATTODAY = 336164623453650944
+    RIBBERTSTIBBERT = 399288392321335307
+    HMMM = 399302648169496577
+    HMMMMM = 399304587745689600
+    YOTSUBATUNES = 399311065416204288
+    NOWIMMOTIVATED = 458107334926008331
+    BIGFOOT = 468234675622641674
+    BIGBOSSU = 468564992334233600
+    OHJEEZ = 543988083306266624
+    PERSONA = 543996812160401408
+    SMIRK = 734835488166379550
+    HMM = 791116665202016256
+    UM = 791118149063147560
+    EDGYSMILE = 796455827556270120
+    CRINGESHOCK = 796455847127416882
+    HEH = 803777598877597736
+    DROOL = 804524633998622740
+    SAYHUH = 832708202852253706
+#endregion
```

Comment:
There's no need to hardcode all of this information because we should be able to get it from calls to `guild.emojis`, `guild.members`, `guild.roles`, `guild.channels`, the `Colour` class, etc. Additionally, all of this information is subject to change, duplicating the info here means any changes to the server will require changes to this file.

Whenever this information is needed, we should get it dynamically whenever possible. There are situations where we may want to get a specific channel or user by name, and in that case the duplication of info is unavoidable (e.g., a hypothetical `foo()` command that posts a message in the "#foo" channel will have to know what channel to look for, so the channel name "#foo" will have to be stored in code or config somewhere in the process), but in other situations a lookup will avoid code changes when the server changes.

For example, if we want a user's color for embeds, instead of hardcoding their color here, we can lookup the user's roles, determine their primary role, and then get the role's color. There may be times we want to get a user by their actual identity (e.g., we want a listener that responds only to James' messages), in which case hardcoding the user ID is pretty unreadable, so a simple map of colloquial names to user IDs would be acceptable and sufficient for those cases - somewhat like the `config.json` dictionary, but just first names to IDs.

Reply:
I agree with majority of this - hard coding this isn't a good look for a scalability. 

User colors was something I wanted to be customizable for each user and not dependent of role, though it's super low priority right now and can be on the back burner. Burning this file to the ground.